cmd/firenear/cli/firehose.go

```diff
@@ -25,12 +25,12 @@ var headTimeDriftmetric = metricset.NewHeadTimeDrift("firehose")
 
 func init() {
 	appLogger := zap.NewNop()
-	logging.Register("github.com/streamingfast/sf-near/firehose", &appLogger)
+	logging.Register("github.com/streamingfast/firehose-near/firehose", &appLogger)
 
 	launcher.RegisterApp(zlog, &launcher.AppDef{
 		ID:          "firehose",
 		Title:       "Block Firehose",
-		Description: "Provides on-demand filtered blocks, depends on common-blocks-store-url and common-blockstream-addr",
+		Description: "Provides on-demand filtered blocks, depends on common-merged-blocks-store-url and common-blockstream-addr",
```

Comment:
`common-blockstream-addr` has not been updated here, it's the old name of `common-live-blocks-addr`